README.md

```diff
@@ -1,2 +1,26 @@
 DBMeter [![Build Status](https://travis-ci.org/DBMeter/DBMeter.svg?branch=master)](https://travis-ci.org/DBMeter/DBMeter)
 =============
+
```

Comment:
We can extract this manual in shell script. It is would be easyer for frequent install. Is'n it?
> sh install.sh

Reply:
Good idea
But I believe for the moment there is almost nothing to automate (just one line of code) and it still will require manual steps (like set other password )